modules/kubernetes/external-dns/charts/external-dns-extras/templates/azure-identity.yaml

```diff
@@ -1,3 +1,4 @@
+{{ if and .Values.resourceID .Values.clientID }}
```

Comment:
I think we should have some kind of standard of telling the modules what provider we are working with. Example: https://github.com/XenitAB/terraform-modules/blob/691da8d4cdd3916e394c0a9f8568f2405f0436cf/modules/kubernetes/velero/variables.tf#L1-L5

Reply:
I do not agree as external-dns works with more than just only cloud providers.